content/docs/ops/security/keys-and-certs/index.md

```diff
@@ -179,3 +181,142 @@ Certificate:
+* If `enableNamespacesByDefault` is `true`, the Citadel instance will create secrets for unlabeled namespaces, otherwise it will not
+
+This logic is captured in the truth table below:
+
+| `ca.istio.io/override` value | `ca.istio.io/env` match | `enableNamespacesByDefault` configuration | Workload secret created |
+|------------------------------|-------------------------|-------------------------------------------|-------------------------|
+|true|yes|true|yes|
+|true|yes|false|yes|
+|true|no|true|yes|
+|true|no|false|yes|
+|true|`unset`|true|yes|
+|true|`unset`|false|yes|
+|false|yes|true|no|
+|false|yes|false|no|
+|false|no|true|no|
+|false|no|false|no|
+|false|`unset`|true|no|
+|false|`unset`|false|no|
+|`unset`|yes|true|yes|
+|`unset`|yes|false|yes|
+|`unset`|no|true|no|
+|`unset`|no|false|no|
+|`unset`|`unset`|true|yes|
+|`unset`|`unset`|false|no|
+
+{{< idea >}}
+When a namespace transitions from `disabled` to `enabled`, Citadel will retroactively generate secrets for all `ServiceAccounts` in that namespace. When transitioning from `enabled` to `disabled`, however, Citadel will not delete the namespace's generated secrets until certificate renewal
+{{< /idea >}}
+
+### Quick examples
+
+#### Deactivating a namespace
+
+For this example, let's create a new sample namespace `foo`
+
+{{< text bash >}}
+$ kubectl create ns foo
+{{< /text >}}
+
+Since service account secrets are created as the default behavior, Citadel should have generated a key/cert secret for the default service account in the `foo` namespace. Let's verify this with
+
+{{< text bash >}}
+$ kubectl get secrets -n foo | grep istio.io
+NAME                    TYPE                           DATA      AGE
+istio.default           istio.io/key-and-cert          3         13s
+{{< /text >}}
+
+Suppose we'd like to prevent Citadel from creating `ServiceAccount` secrets in target namespace `foo`. This can be done through labeling the namespace with
+
+{{< text bash >}}
+$ kubectl label ns foo ca.istio.io/override=false
+{{< /text >}}
+
+Now, if we create a new `ServiceAccount` in this namespace with
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sample-service-account
+  namespace: foo
+EOF
+{{< /text >}}
+
+and then check the namespace secrets again
+
+{{< text bash >}}
+$ kubectl get secrets -n foo | grep istio.io
+NAME                    TYPE                           DATA      AGE
+istio.default           istio.io/key-and-cert          3         11m
+{{< /text >}}
+
+We can observe that no new `istio.io/key-and-cert` secret was generated for the `sample-service-account` service account.
+
+#### Opt-in Citadel secret generation
+
+Let's suppose that as an operator, we would like to make `ServiceAcount` secret generation opt-in (i.e. don't generate secrets unless otherwise specified). In this case we should set the following in our mesh configuration
```

Comment:
You mean helm chart configuration right? `MeshConfig` specificailly refer to something else, being more specific can avoid this confusion.